Makefile

```diff
@@ -24,6 +26,7 @@ ADDONS_DIR := addons
 YTT_TESTS_DIR := pkg/v1/providers/tests
 PACKAGES_SCRIPTS_DIR := $(abspath hack/packages/scripts)
 UI_DIR := pkg/v1/tkg/web
+GO_MODULES=$(shell find . -path "*/go.mod" | xargs -I _ dirname _)
```

Comment:
```
$ find . -path "*/go.mod" | xargs -I _ dirname _
.
./addons
./addons/pinniped/post-deploy
./hack/tools
./hack/packages/kbld-image-replace
./pinniped/generated/1.18/apis
./pinniped/generated/1.18/client
./pinniped/generated/1.20/apis
./pinniped/generated/1.20/client
./pinniped/generated/1.19/apis
./pinniped/generated/1.19/client
./pinniped/generated/1.17/apis
./pinniped/generated/1.17/client
./pinniped
./pkg/v1/providers/tests
```

This seems to included all the `generated/*` modules from the cloned pinniped repo. Should we filter these out?

Reply:
Ah, good point. Probably not. Let me filter those out quick...